autoload/lsp/ui/vim/output.vim

```diff
@@ -1,18 +1,174 @@
+let s:supports_floating = exists('*nvim_open_win') || has('patch-8.1.1517')
+let s:win = v:false
```

Comment:
can we rename this to `winid`. It is confusing with `windows` OS.

Reply:
done